app/src/main/java/com/imashnake/animite/features/media/MediaPage.kt

```diff
@@ -160,7 +160,7 @@ fun MediaPage(
                     )
                 }
 
-                if (media.characters != null) {
+                if (!media.characters.isNullOrEmpty()) {
```

Comment:
Should we be guarding against this elsewhere? Either make `media.characters` non-null, or ensure when it's not null it's always populated

Reply:
I'll try to see if there's some way of dealing with this using Apollo Kotlin.